_rules/descriptive-labels-f0c5c5.md

```diff
@@ -0,0 +1,280 @@
+---
+id: f0c5c5
+name: Form field label is descriptive
+rule_type: atomic
+description: |
+  This rule checks that labels describe the purpose of form field elements.
+accessibility_requirements: 
+  wcag21:2.4.6: # Headings and labels (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
```

Comment:
We might want to include "Language" here also, following #671

Reply:
@carlosapaduarte, I agree, but no decision has been made om #671, and I guess we should at one point stop adding new things to this PR and just see if we can get it closed off soon :) 
I am imagining that #671, if agreed on, will  need to update all the rules that are impacted by the decision, and we might as well do it in one go.